### ISSUE TITLE
Make gRPC usable without this package

### ISSUE THREAD

**Issue by mcilloni** [stanley-cheung/Protobuf-PHP#19]
The protobuf main repository now includes [a php implementation](https://github.com/google/protobuf/tree/master/php). Does that make this repository obsolete? Is that implementation still unfeasible to be used with gRPC, given that both Docker images and the standard documentation still use this repository as their PHP protobuf compiler?

**Comment by stanley-cheung** [comment]
Yes, we will be moving to the official google/protobuf implementation with the gRPC 1.1 release, which will happen very soon. This repo will be obsolete after that switch is done.

**Comment by mcilloni** [comment]
Perfect, this is really a good news. Thanks a lot for the hard work on PHP gRPC!
